lasagne/init.py

```diff
@@ -100,9 +100,9 @@ def sample(self, shape):
 
 
 class Glorot(Initializer):
-    """Glorot weight initialization [1]_.
+    """Glorot weight initialization.
 
-    This is also known as Xavier initialization.
+    This is also known as Xavier initialization[1]_.
```

Comment:
+1 for moving the reference (to fix the autosummary), but you're missing a blank before the `[1]_`.